manifests/init.pp

```diff
@@ -100,7 +100,7 @@
   }
   concat::fragment { 'config.yaml > except packages':
     target  => "${install_path}/config.yaml",
-    content => template('verdaccio/config.yaml.erb'),
+    content => template($conf_template),
```

Comment:
Where the `'verdaccio/config.yaml.erb'` default value will be defined now?

Reply:
It's been defined in the top of the class, it's just never been used until now. I'm guessing it was just an oversight.

https://github.com/verdaccio/puppet-verdaccio/blob/60c9caa1e26125f2f50c171b8139d96bdaacb86a/manifests/init.pp#L34